Kudu.Services/Jobs/JobsController.cs

```diff
@@ -68,7 +74,7 @@ public HttpResponseMessage EnableContinuousJob(string jobName)
         {
             try
             {
-                _continuousJobsManager.EnableJob(jobName);
+                _continuousJobsManagers.GetWriteManager(_settings).EnableJob(jobName);
```

Comment:
the code in this controller is error prone, but each API needs to get a manager depending on the API scenario. List APIs combine both, write APIs need to call `GetWriteManager`, other APIs call `GetManagerForJob()` which will look into the primary manager first, then the secondary